rpcs3/rpcs3qt/main_window.cpp

```diff
@@ -295,12 +295,14 @@ bool main_window::OnMissingFw()
 				"\n<br>For information about how to obtain the required firmware read the <a href=\"https://rpcs3.net/quickstart\">quickstart guide</a>.");
 
 	QMessageBox* mb = new QMessageBox(QMessageBox::Question, title, message, QMessageBox::Ok | QMessageBox::Cancel, this, Qt::Dialog | Qt::MSWindowsFixedSizeDialogHint | Qt::WindowStaysOnTopHint);
-	mb->deleteLater();
```

Comment:
This shouldn't be an issue, as the object will only be deleted if we come back to the encompassing event loop, which isn't the same event loop as the mb->exec

Reply:
If you're paranoid about this, I'd rather have it on the stack in the first place.